src/anomalib/models/image/patchcore/lightning_model.py

```diff
@@ -139,3 +138,11 @@ def configure_transforms(self, image_size: tuple[int, int] | None = None) -> Tra
                 Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
             ],
         )
+
+    def default_post_processor(self) -> OneClassPostProcessor:
```

Comment:
wouldn't it be possible to just set the post-processor here instead of using default post-processor. Not sure if i get the logic?

Reply:
see my previous comment